web/struct_tree_layer_builder.js

```diff
@@ -82,8 +83,11 @@ class StructTreeLayerBuilder {
 
   #elementAttributes = new Map();
 
-  constructor(pdfPage) {
+  #viewBox;
+
+  constructor(pdfPage, viewBox) {
     this.#promise = pdfPage.getStructTree();
+    this.#viewBox = viewBox;
```

Comment:
Given what we're passing in, shouldn't this be named as follows?
```suggestion
  constructor(pdfPage, rawDims) {
    this.#promise = pdfPage.getStructTree();
    this.#rawDims = rawDims;
```